schematic/models/validate_manifest.py

```diff
@@ -295,8 +298,7 @@ def validate_manifest_values(
         warnings = []
         col_attr = {}  # save the mapping between column index and attribute name
 
-        # Replace nans with empty strings so jsonschema
-        manifest = manifest.replace({np.nan: ""})
+        manifest = convert_nan_entries_to_empty_strings(manifest=manifest)
 
         # numerical values need to be type string for the jsonValidator
```

Comment:
Not in this ticket, but I think this step of converting numerical values to string should be called out in a separate function just like convert_nan_entries_to_empty_strings. That will make this part more obvious and easier to test.